assets/js/home.js

```diff
@@ -158,188 +164,188 @@ $(document).ready(function(){
+        selectedDate.innerHTML = e.currentTarget.textContent
+        // Find index to the right slide
+        const category = selectedDate.closest('.bubbleContainer').getAttribute('data-category')
+        let indexRightSlide = document.querySelector('.newsfeed.' + category + ' .slickContainer .cardContainer[data-content-year="' + e.currentTarget.textContent + '"]').closest(".slick-slide").getAttribute('data-slick-index')
+        $('.newsfeed.' + category + ' .slickContainer').slick('slickGoTo', indexRightSlide)
+        // if All Select Selected
+        const isAllSelected = document.querySelector('.bubbleContainer.all').classList.contains('active')
+        if (isAllSelected) {
+          for (let choice of choices) {
+            choice.classList.add('active')
           }
-        })
-      }
-    })
-  }
+        }
+      })
+    }
+  })
+}
 
-  // News link with feed's publications
-  const newsList = document.querySelectorAll('.newsWrapper[data-content-id]')
-  const allFeedCta = document.querySelector('.bubbleContainer.all')
-  
-  for (let newsElt of newsList) {
-    newsElt.addEventListener('click', function(e) {
+// News link with feed's publications
+const newsList = document.querySelectorAll('.newsWrapper[data-content-id]')
+const allFeedCta = document.querySelector('.bubbleContainer.all')
 
-      for (let newsElt of newsList) {
-        newsElt.classList.remove('active')
-      }
-      newsElt.classList.add('active')
-      newsId = newsElt.getAttribute('data-content-id');
+for (let newsElt of newsList) {
+  newsElt.addEventListener('click', function (e) {
 
-      // 1 - click on allfeed CTA
-      activateNewsFeed('all')
-      for (let choice of choices) {
-        choice.classList.add('active')
-      }
+    for (let newsElt of newsList) {
+      newsElt.classList.remove('active')
+    }
+    newsElt.classList.add('active')
+    newsId = newsElt.getAttribute('data-content-id');
 
-      // 2 - Save All feed on localStorage
-      localStorage.setItem('feed', 'all');
+    // 1 - click on allfeed CTA
+    activateNewsFeed('all')
+    for (let choice of choices) {
+      choice.classList.add('active')
+    }
 
-      // 2 - find data-content-id on newsfeed all element and calculate index position of this slide
-      let indexRightSlide = document.querySelector('.newsfeed.all .slickContainer .cardContainer[data-content-id="' + newsId + '"]').closest(".slick-slide").getAttribute('data-slick-index')
+    // 2 - Save All feed on localStorage
+    localStorage.setItem(email + '-feed', 'all');
 
-      // 3 - go to this slide
-      $('.newsfeed.all .slickContainer').slick('slickGoTo', indexRightSlide)
+    // 2 - find data-content-id on newsfeed all element and calculate index position of this slide
+    let indexRightSlide = document.querySelector('.newsfeed.all .slickContainer .cardContainer[data-content-id="' + newsId + '"]').closest(".slick-slide").getAttribute('data-slick-index')
```

Comment:
personnaly, i would have done a single object containing a array of settings. and for each settings object, a user key a the top level (an id, instead of an email), then all the other keys in sub level.

ex:  object "settings" :
{
  "123": [
    {
	  "email": "xxx@xxxxxxx.xx"
      "feed": "press",
      "slidePress": 2,
      "slidePub": 0,
      "slideTalk": 0
    },
	"456": [
    {
	  "email": yyyy@yyyyyyyy.yyy"
      "feed": "talk",
      "slideAll": 6,
      "slidePub": 0,
      "slideTalk": 0
    }
}

Reply:
ok i try do to this :) At the begin i want to make KISS, but now it's interessting with like that 👍